src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
+    private int number;
+
+    public Ball(Colors color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String toString() {
+        return "The winner is a " + color + " ball with a number - " + number + ";";
```

Comment:
Any Ball is the winner? :)

Reply:
I didn't understand here

Reply:
he means that not any ball from the lottery is the winner ball